### DISCUSSION TITLE
make it easier to review translations in the pull request

### DISCUSSION THREAD

**Discussion by moodysalem** [crowdin/github-action#184] [General]
**Is your feature request related to a problem? Please describe.**
i would like to see just the new translations or translation changes in the pull request for downloaded translations, rather than all the context changes

**Describe the solution you'd like**
would be great if the default pr description offered a summary of the changes

**Describe alternatives you've considered**
n/a

**Additional context**
just context changes:
<img width="229" alt="image" src="https://github.com/crowdin/github-action/assets/7897876/0fec1a28-c60a-4ff7-919a-3adca84c70d0">

**Comment by andrii-bodnar** [comment]
@moodysalem could you please follow the "Feature Request" template for suggesting enhancements? 

Also, please describe in more detail what you mean by "summary of changes".

Thank you.

**Comment by moodysalem** [comment]
> Also, please describe in more detail what you mean by "summary of changes".

<img width="923" alt="image" src="https://github.com/crowdin/github-action/assets/7897876/39aa7500-d98a-4664-83ba-02762cee8b44">
I moved some code around without changing any copy, extracted and uploaded the translations automatically via GH actions, then the download github action sent this PR with +/-2705 LOC for all the new context line numbers in the translation files

Really difficult for me to figure out based on this what's actually being downloaded

**Comment by andrii-bodnar** [comment]
The GH Action uses the [CLI](https://crowdin.github.io/crowdin-cli/) which uses the [Build Project Translation API](https://developer.crowdin.com/api/v2/#operation/api.projects.translations.builds.post) to download translation.

It returns just a ZIP archive with the translation files and there is no meta-information about the downloaded translations.

**Reply by moodysalem** [reply]
The github action could summarize the changes contained in the git diff for all the .po files

For example:

```
100 translations had their context updated


## es.po
### New translations:
`Connect wallet` -> `Conecte monedaro`

### Updated translations:
`Conecte monedaro` -> ``
...
```
and so on, in a markdown table for example

the github pr description is currently wasted if it only contains some random fixed description
